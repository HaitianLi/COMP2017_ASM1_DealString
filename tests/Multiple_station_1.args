Melbourne Sydney 09:32:21
