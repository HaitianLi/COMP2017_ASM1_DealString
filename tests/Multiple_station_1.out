The next train to Sydney from Melbourne departs at 10:05:00
